aws/src/test/java/org/apache/iceberg/aws/TestAwsProperties.java

```diff
@@ -97,4 +97,24 @@ public void testS3MultipartThresholdFactorLessThanOne() {
         () -> new AwsProperties(map));
   }
 
+  @Test
+  public void testS3FileIoDeleteBatchSizeTooLarge() {
+    Map<String, String> map = Maps.newHashMap();
+    map.put(AwsProperties.S3FILEIO_DELETE_BATCH_SIZE, "2000");
+    AssertHelpers.assertThrows("should not accept batch size greater than 1000",
+            IllegalArgumentException.class,
```

Comment:
the indentations in tests seem to be off? I think it should have 4 indent

Reply:
Updated. Strange, I think when I was working on adding these tests, my IDE was using different the default indentation settings rather than the Iceberg ones.